cdk/bin/cdk.ts

```diff
@@ -4,7 +4,7 @@ import * as cdk from '@aws-cdk/core';
 import { CdkStack } from '../lib/cdk-stack';
 
 const app = new cdk.App();
-new CdkStack(app, `earthdata-dashboard-${process.env.PROJECT}-${process.env.STAGE}`, {
+new CdkStack(app, `earthdata-dashboard-${process.env.STAGE}`, {
```

Comment:
why did you remove PROJECT here?

Reply:
I just pulled in the changes from [here](https://github.com/MAAP-Project/earthdata-dashboard-maap/blob/staging/cdk/bin/cdk.ts). Was that not supposed to be pulled? @philvarner

Reply:
I think that was an inadvertent change that got into my PR. It should be removed from this PR, and then I also need to fix it in earthdata-dashboard-maap to be `earthdata-dashboard-maap-${process.env.STAGE}` instead of `earthdata-dashboard-${process.env.STAGE}`

Glad we found this!

Reply:
PR -- https://github.com/MAAP-Project/earthdata-dashboard-maap/pull/15